generators/cypress/templates/src/test/javascript/cypress/integration/account/register-page_spec.ts.ejs

```diff
@@ -0,0 +1,102 @@
+import {
+    usernameRegisterSelector,
+    emailRegisterSelector,
+    firstPasswordRegisterSelector,
+    secondPasswordRegisterSelector,
+    submitRegisterSelector,
+    classInvalid,
+    classValid } from '../../support/commands';
```

Comment:
indentation could be better 😇

Reply:
This indentation have been taken from the generated file, so with prettier applied ;)